internal/services/mysql/mysql_flexible_server_resource.go

```diff
@@ -335,6 +338,9 @@ func resourceMysqlFlexibleServer() *pluginsdk.Resource {
 		),
 	}
 
+	// TODO When using >= v0.71.1 of go-azure-helpers, remove next line and update the schema to use `OptionalComputed` zone validation
```

Comment:
Is this change needed to get `public_network_access` out the door? We should do separate PRs here and since it looks like we're about to update go-azure-helpers, can we instead skip this and make the change later?

Reply:
Not functionally needed, but it allows the `basic` and `publicNetworkAccess` acceptance tests to pass.